recipes/mender-client/debian-master/rules

```diff
@@ -3,40 +3,33 @@
 %:
```

Comment:
:warning: I cannot see the recipes for the `3.5.x` series. I thought I had fixed it! Either I made a mistake or the Git merge here is doing something strange. I will look into it and amend if necessary.

Reply:
Okey, the was done in `master` branch, see:
* https://github.com/mendersoftware/mender-dist-packages/commit/0a47cf999e333a7961ff900af5b30c61dae8a12a